r-package/grf/DESCRIPTION

```diff
@@ -1,20 +1,21 @@
 Package: grf
-Title: Generalized Random Forests (Beta)
+Title: Generalized Random Forests
 Version: 0.10.3
-Author: Julie Tibshirani [aut, cre],
-    Susan Athey [aut],
-    Rina Friedberg [ctb],
-    Vitor Hadad [ctb],
-    Luke Miner [ctb],
-    Stefan Wager [aut],
-    Marvin Wright [ctb]
+Authors@R:c(
+    person("Julie", "Tibshirani", role = c("aut", "cre"), email = "jtibs@cs.stanford.edu"),
```

Comment:
Could we omit my email here?

Reply:
I think CRAN requires one email: 

```
Authors@R field gives no person with maintainer role, valid email
address and non-empty name.

See section 'The DESCRIPTION file' in the 'Writing R Extensions'
manual.
```

replace it with another?